numpyro/hmc_util.py

```diff
@@ -241,6 +245,12 @@ def init_fn(z, rng, step_size=1.0, inverse_mass_matrix=None, mass_matrix_size=No
                 inverse_mass_matrix = np.ones(mass_matrix_size)
             else:
                 inverse_mass_matrix = np.identity(mass_matrix_size)
+            mass_matrix_sqrt = inverse_mass_matrix
+        else:
+            if diag_mass:
```

Comment:
@fehiepsi - what do you think about changing `diag_mass=True` to `dense_mass=False`? I don't feel too strongly about it, but it seems a bit odd to me to specify `diag_mass=False` to estimate the full mass matrix.